service/api_patch.go

```diff
@@ -319,6 +129,12 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	githubOauthToken, err := evergreen.GetEnvironment().Settings().GetGithubOauthToken()
```

Comment:
the settings object is a field on the api server instance.

Reply:
done